osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -417,5 +438,31 @@ public PanelBackground()
                 Beatmap.BindValueChanged(beatmap => backgroundSprite.Beatmap.Value = beatmap.NewValue);
             }
         }
+
+        private class OwnerAvatar : UpdateableAvatar, IHasTooltip
+        {
+            public OwnerAvatar()
+            {
+                AddInternal(new TooltipArea(this)
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Depth = -1
+                });
+            }
+
+            public LocalisableString TooltipText => User == null ? "loading user..." : $"queued by {User.Username}";
```

Comment:
The "loading user" text appears incorrectly in some contexts. For instance, the screenshot below was taken during creation of a new multiplayer room:

![osu_2021-11-25_20-39-46](https://user-images.githubusercontent.com/20418176/143493859-6d604959-61ec-420a-8d8c-f2de9d276a88.jpg)